include/fiwix/bios.h

```diff
@@ -20,6 +20,7 @@ struct bios_mem_map {
 	int type;
 };
 extern struct bios_mem_map bios_mem_map[NR_BIOS_MM_ENT];
+extern struct bios_mem_map orig_bios_mem_map[NR_BIOS_MM_ENT];
```

Comment:
If there will not be a completely different `bios_mem_map()` function, what's the point to rename it to `orig_bios_mem_map()`?

I mean, if there is only one function doing the work, why this change in its name?

Its original name already indicates that this is the original BIOS memory map.